packages/editor/src/components/media-placeholder/index.js

```diff
@@ -127,7 +127,7 @@ class MediaPlaceholder extends Component {
 					{ __( 'Upload' ) }
 				</FormFileUpload>
 				<MediaUpload
-					gallery={ multiple }
+					gallery={ multiple && type != "audio" }
```

Comment:
I'd say, it's probably better if the check is `multiple && type === 'image'`

Reply:
Yes, I agree.

And, actually, there should be two different lines, one for `gallery` and one for `playlist`, like this.

```
gallery={ multiple && type === 'image' }
playlist={ multiple && type === 'audio' }
```